web-app/src/app/services/feeds/index.ts

```diff
@@ -0,0 +1,201 @@
+import createClient, { type Middleware } from 'openapi-fetch';
+import type { paths } from '../feeds/types';
```

Comment:
[nit] import could be `import type { paths } from './types'`

Reply:
fixed